static/menu.js

```diff
@@ -0,0 +1,18 @@
+const navigationBar = $("#navbar");
+
+$("#contentBox").css("top", $("#navbar").css("height")); //Sets the content below the navbar.
+
+document.onscroll = () => {
```

Comment:
We should probably have a look at adding babel soon, so that we can use es6 stuff. That would tidy up the ticket code a bit as well. I just didn't want to add that when the entire project had a total of ~160 lines of js.